database/snowflake/README.md

```diff
@@ -1,10 +1,14 @@
 # Snowflake
 
-`snowflake://user:password@accountname/schema/dbname?query`
+`snowflake://user:password@accountname/schema/dbname/warehouse?query`
```

Comment:
warehouse should be a query param: https://godoc.org/github.com/snowflakedb/gosnowflake#hdr-Connection_Parameters

Reply:
Done